test/browser/step_definitions/CICCheckAnswersNavigation.step.js

```diff
@@ -14,7 +14,7 @@ Given(
     const cdPage = new CheckDetailsPage(await this.page);
 
     expect(await cdPage.isCurrentPage()).to.be.true;
-  }
+  },
```

Comment:
Hmm I don't think this formatting change is actually correct? What command did you run? Maybe @Z-Mohamoud can give us a hand?

Reply:
I ran yarn lint and yarn prettier:fix